src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -67,9 +67,25 @@
         - name: --windows-admin-username
           type: string
           short-summary: User account to create on windows node VMs.
+          long-summary: |-
+            Rules for windows-admin-username:
+                - restriction: Cannot end in "."
+                - Disallowed values: "administrator", "admin", "user", "user1", "test", "user2", "test1", "user3", "admin1", "1", "123", "a", "actuser", "adm", "admin2", "aspnet", "backup", "console", "david", "guest", "john", "owner", "root", "server", "sql", "support", "support_388945a0", "sys", "test2", "test3", "user4", "user5".
```

Comment:
do we have public doc about this disallowed value? if yes, can we link the doc here?